src/legacy/core_plugins/console/server/api_server/spec/generated/indices.get.json

```diff
@@ -1,6 +1,7 @@
 {
   "indices.get": {
     "url_params": {
+      "include_type_name": "__flag__",
```

Comment:
This gives me an error. According to the docs it is deprecated in 7.0

<img width="1308" alt="Screen Shot 2020-01-14 at 16 56 46" src="https://user-images.githubusercontent.com/2854616/72340874-f7755680-36ee-11ea-9563-f92f2ee93e5b.png">

<img width="918" alt="Screen Shot 2020-01-14 at 16 56 34" src="https://user-images.githubusercontent.com/2854616/72340889-00febe80-36ef-11ea-8fd5-edd365259caf.png">

Reply:
Hmm, perhaps it is best to remove these for now, in the event that it results in an error.